lib/helpers.js

```diff
@@ -34,51 +58,82 @@ function buildMessage (data) {
   })
 
   if (blocked.length || needReview.length || inProgress.length) {
-    let message = []
+    const total = blocked.length + needReview.length + inProgress.length
+    let message = askingForOwnAssignedPrs ? [`*${total} issue(s) need your attention*`] : []
 
-    if (blocked.length)
+    if (blocked.length) {
       message = [
         ...message,
-        messages.PR_LIST_BLOCKED.replace(/\${NUMBER}/, blocked.length),
+        '\n',
+        `:warning: *${blocked.length}* PRs are *blocked*`,
         ...orderPRsByRepo(blocked)
       ]
+    }
 
-    if (needReview.length)
+    if (needReview.length) {
       message = [
         ...message,
-        messages.PR_LIST_REVIEW.replace(/\${NUMBER}/, needReview.length),
+        '\n',
+        `:eyes: *${needReview.length}* PRs need a review:`,
         ...orderPRsByRepo(needReview)
       ]
+    }
 
-    if (inProgress.length)
+    if (inProgress.length) {
       message = [
         ...message,
-        messages.PR_LIST_WIP.replace(/\${NUMBER}/, inProgress.length),
+        '\n',
+        `:construction_worker: *${inProgress.length}* PRs are in-progress:`,
         ...orderPRsByRepo(inProgress)
       ]
+    }
+
+    if (!askingForOwnAssignedPrs) {
+      // tally up and rank who has the most items assigned to themselves
+      message = [
+        ...message,
+        '\n',
+        '*Pull requests assigned per user:*'
+      ]
+      const assigneePrCount = {}
+      whitelisted.forEach(pr => {
+        if (!pr.assignee || !pr.assignee.login) return
```

Comment:
Yes, but my point was that it would be helpful in this summary view to have a quick indication of whether there are any "hanging" PRs that don't have anyone assigned to take care of them.

Reply:
Isn't that accounted for already for in the list of PRs with the 🤷‍♀️ icon?